src/sql/workbench/contrib/notebook/browser/notebook.component.html

```diff
@@ -9,35 +9,13 @@
 	</div>
 	<div #container class="scrollable" style="flex: 1 1 auto; position: relative; outline: none" (click)="unselectActiveCell()" (scroll)="scrollHandler($event)">
 		<loading-spinner [loading]="isLoading"></loading-spinner>
-		<div class="hoverButtonsContainer" *ngIf="(cells && cells.length > 0) && !isLoading">
```

Comment:
Let's double check with PM what we should ship in April, and whether these really should be removed.

Reply:
@yualan and I chatted. He's fine with leaving these out.